libraries/botbuilder-dialogs-adaptive-teams/src/actions/actionHelpers.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @module botbuilder-dialogs-adaptive-teams
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { ExpressionProperty } from 'adaptive-expressions';
+import { DialogContext } from 'botbuilder-dialogs';
+
+/**
+ * Get the value of the string expression from the Dialog Context.
+ *
+ * @param {DialogContext} dc A DialogContext object.
+ * @param {ExpressionProperty<any>} expressionProperty The expressionProperty to use to retrieve a value from the DialogContext.
+ * @returns {string} The value of the evaluated stringExpression.
+ */
+export function getValue<T>(dc: DialogContext, expressionProperty: ExpressionProperty<T>): T {
```

Comment:
After enabling `strict` mode you'll have to annotate methods that return `nil` or `undefined` like `T | undefined` or `T | nil`.